pam_yubico.c

```diff
@@ -410,21 +410,25 @@ display_error(pam_handle_t *pamh, const char *message) {
   pmsg[0] = &msg[0];
   msg[0].msg = (char *) message; /* on some systems, pam_message.msg isn't const */
   msg[0].msg_style = PAM_ERROR_MSG;
-  retval = conv->conv(1, pmsg, &resp, conv->appdata_ptr);
-
-  if (retval != PAM_SUCCESS) {
-    D(("conv returned error: %s", pam_strerror (pamh, retval)));
+  if(conv && conv->conv){
```

Comment:
Do you want to update the pull request to that (check before msg and return early) or should I just commit it?